pkg/agent/multicluster/stretched_networkpolicy_controller_test.go

```diff
@@ -45,8 +45,8 @@ import (
 )
 
 const (
-	interval = time.Millisecond
-	timeout  = time.Second
+	interval = 10 * time.Millisecond
+	timeout  = 5 * time.Second
```

Comment:
Can it be shorter? considering it's just a fake client and cache.

Reply:
Done.